packages/oats-runtime/package.json

```diff
@@ -23,6 +23,8 @@
   },
   "dependencies": {
     "@smartlyio/safe-navigation": "^5.0.1",
+    "@types/escape-html": "^1.0.1",
+    "escape-html": "^1.0.3",
```

Comment:
this has been last updated 2019. I feel that we could maybe do without the dependency

Reply:
i also thought so, but couldn't find anything. Last publish was 7 years ago right, but maybe it's just perfect? 😅
i mean it doesn't do anything but escapes quotes and ampersands

Reply:
we can do it ourselves of course:
```
function escape(htmlStr) {
   return htmlStr.replace('&', "&amp;")
         .replaceAll('<', "&lt;")
         .replaceAll('>', "&gt;")
         .replaceAll('"', "&quot;")
         .replaceAll("'", "&#39;");        

}
```

Reply:
you wanna deal with left-pad all again 😬

Reply:
sure, i'll remove the anchor
i can even add `save-exact=true` to .npmrc